scripts/run-once-scripts/postgres-migration/batch-delete-dynamo-items.ts

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
This is just the script that was used for `delete-messages.ts`. I moved it here so it could be reusable.

.